src/pages/en/guides/deploy.md

```diff
@@ -1,523 +1,83 @@
+1. Add your repository to an online Git provider (e.g. in GitHub, GitLab, Bitbucket)
 
-1. Set the [`site`](/en/reference/configuration-reference/#site) and, if needed, [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
-    - `site` should be something like `https://<YOUR USERNAME>.github.io/`
-    - `base` should be your repository’s name. (If your repository is named `<YOUR USERNAME>.github.io`, you don’t need to include `base`.)
-1. Create a new file in your project at `.github/workflows/deploy.yml` and paste in the YAML below.
+1. Choose a host that supports **continuous deployment** (e.g. Netlify, Vercel) and import your Git repository as a new site/project.
 
-    ```yaml
-    name: Github Pages Astro CI
+    Many common hosts will recognize your project as an Astro site, and should choose the appropriate configuration settings to build and deploy your site as shown below. (If not, these settings can be changed.)
 
-    on:
-      # Trigger the workflow every time you push to the `main` branch
-      # Using a different branch name? Replace `main` with your branch’s name
-      push:
-        branches: [main]
-      # Allows you to run this workflow manually from the Actions tab on GitHub.
-      workflow_dispatch:
+    :::note[Deploy Settings]
+    - **Build Command:** `astro build` or `npm run build`
+    - **Publish directory:** `dist`
+    :::
 
-    jobs:
-      deploy:
-        runs-on: ubuntu-20.04
+1. Click "Deploy" and your new website will be created at a unique `.netlify.app` or `.vercel.app` URL.
 
-        # Allow this job to push changes to your repository
-        permissions:
-          contents: write
 
-        steps:
-          - name: Check out your repository using git
-            uses: actions/checkout@v2
+Your site will be automatically configured to watch your Git provider's main branch for changes, and to rebuild and republish at each new commit. These settings can typically be configured in your host provider's dashboard UI.
 
-          - name: Use Node.js 16
-            uses: actions/setup-node@v2
-            with:
-              node-version: 16
+### CLI Deployment
 
-          # Not using npm? Change `npm ci` to `yarn install` or `pnpm i`
-          - name: Install dependencies
-            run: npm ci
+Some hosts will have their own CLI you can install globally to your machine using npm.
 
-          # Not using npm? Change `npm run build` to `yarn build` or `pnpm run build`
-          - name: Build Astro
-            run: npm run build
+1. Install your host's CLI globally, for example:
 
-          - name: Deploy to GitHub Pages
-            uses: peaceiris/actions-gh-pages@v3
-            with:
-              github_token: ${{ secrets.GITHUB_TOKEN }}
-              # `./dist` is the default Astro build directory.
-              # If you changed that, update it here too.
-              publish_dir: ./dist
+    ```bash
+    npm install netlify-cli -g
```

Comment:
Nit but this way it reads more like English left to right. Even if personally I’d use `npm i -g` for speed, this is easier for beginners to understand what’s happening I think.

```suggestion
    npm install --global netlify-cli
```

Reply:
Then your nit is with Netlify!  ;) Happy to improve readability!